Frontend/src/pages/MyRecipes.tsx

```diff
@@ -1,152 +1,156 @@
+                                        <li>250 g Ramen-Nudeln</li>
+                                        <li>1 EL Sesamsamen</li>
+                                    </ul>
+                                </div>
+                            </div>
+                        </Item>
+                        <Item>
+                            <div className="Anleitung">
+                                <h2>Anleitung:</h2>
+                                <ol>
+                                    <li><strong>Gemüse vorbereiten:</strong> Pilze vierteln; Ingwer und Knoblauch fein
+                                        hacken. Pak Choi in Streifen und Frühlingszwiebel in Ringe schneiden.
+                                    </li>
+                                    <li><strong>Eier kochen:</strong> Eier 6 Minuten kochen, dann pellen und halbieren.
+                                    </li>
+                                    <li><strong>Anbraten:</strong> Sesamöl in einem Topf erhitzen. Ingwer und Knoblauch
+                                        anbraten.
+                                    </li>
+                                    <li><strong>Brühe zubereiten:</strong> Pilze, Misopaste, Sojasauce, Apfelessig und
+                                        Gemüsebrühe in den Topf geben. Zum Kochen bringen und 5 Minuten köcheln lassen.
+                                    </li>
+                                    <li><strong>Nudeln hinzufügen:</strong> Ramen-Nudeln und Pak Choi hinzufügen, 2 Minuten
+                                        kochen.
+                                    </li>
+                                    <li><strong>Servieren:</strong> Suppe in Schüsseln verteilen. Je eine halbe Ei
+                                        hinzufügen, mit
+                                        Frühlingszwiebelringen und Sesamsamen garnieren.
+                                    </li>
+                                </ol>
+                            </div>
+                        </Item>
+                    </Stack>
+                    <div>
+
+                    </div>
+
+                    {/*    <div className="FAB">*/}
+                    {/*        <a className="AdvBtn" href={'/adventure'}>*/}
+                    {/*            <img src={BtnAdvntrz} className="AdvBtn_self" alt="AdventurizeIt"/>*/}
+                    {/*        </a>*/}
+                    {/*    </div>*/}
+                    {/*    <Button className="MakeShoppingList" variant="contained" color="success">*/}
+                    {/*        Download <br/>*/}
+                    {/*        Shopping List*/}
+                    {/*    </Button>*/}
+                    {/*</div>*/}
+                    {/*<div id="StickyCards_Container">*/}
+                    {/*     <div className="StickyNoteCont">*/}
+                    {/*     <img className="StickyNote" src={StickyNote} alt="feltDown"/>*/}
+                    {/*     </div>*/}
```

Comment:
why is this commented out? can this be removed?

Reply:
The page isn't finished yet. So it's unclear if this will be part of the design.